src/scss/components/button/_base.scss

```diff
@@ -1,6 +1,9 @@
+$btn-base-pd-vertical: $pd--s;
+$btn-base-pd-horizontal: $pd--m - ($fz--m/2);
```

Comment:
I don't agree with this. It is squeezing all buttons, just in order to fix the circle one.
Why not adding a specific horizontal padding only on `.Btn--circle`?